test/truffle/ecosystem/blog/Gemfile

```diff
@@ -21,14 +21,14 @@ gem 'sqlite3'
 # Use Puma as the app server
 gem 'puma', '~> 3.11'
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
+gem 'sass-rails', '~> 6.0'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'mini_racer', platforms: :ruby
 
 # Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.2'
+gem 'coffee-rails'#, '~> 4.2'
```

Comment:
Should this still have a version constraint?  What does a new Rails 6 app use?

Reply:
Good point. It looks like new Rails 6 app doesn't even have coffee as a part of Gemfile, I assume because of the webpack infra.

Should I delete this line altogether?

Reply:
Yes please, we'd like to be as close as possible to a vanilla Rails 6 app

Reply:
Updated the PR 👍